app/components/listing/Services.jsx

```diff
@@ -3,14 +3,14 @@ import ReactMarkdown from 'react-markdown';
 import DetailedHours from './DetailedHours';
 import Notes from './Notes';
 
-const Services = ({ services }) => services && services.length > 0
-      && (
-        <ul className="service--section--list">
-          {services.map(service => (
-            <Service service={service} key={service.id} />
-          ))}
-        </ul>
-      );
+const Services = ({ services }) => (services?.length > 0
+  ? (
+    <ul className="service--section--list" data-cy="service-list">
+      {services.map(service => (
+        <Service service={service} key={service.id} />
+      ))}
+    </ul>
+  ) : null);
```

Comment:
Personal preference. It is more explicit and declarative, I don't like that a statement with a `&&` leads to any return other than a boolean in Javascript, always felt like an anti-pattern to me.

Not that religious about it though, happy to go back to the previous implementation if you prefer it.

Reply:
I have a weak preference for the opposite, especially because it's "blessed" by the official React docs, but I also don't care that strongly about it, as long as we're consistent. No need to change this here right now, since I think the TypeScript conversion is a much more important step forward.